### DISCUSSION TITLE
Improvements on the bot

### DISCUSSION THREAD

**Discussion by fabianaferreira** [Lucas-Cerqueira/carona-bot-js#41] [Ideas]
Discussing with other alumni from UFRJ, we discovered that there is another bot, but written in PHP, which lost part of the compatibility due to the end of free dynos on Heroku. Therefore, we started this discussion to share the ideas, improvements, or features that the other project has so that we can incorporate them into this project.

**Comment by teredeby** [comment]
Drivers usually don't have 'ida' and 'volta' active at the same time. It would be helpful to have an '/lotei' command that changes the status of the active 'carona' with no need to specify if it was a 'ida' or a 'volta'
